datafusion/core/src/datasource/physical_plan/json.rs

```diff
@@ -374,17 +376,19 @@ mod tests {
     async fn test_additional_stores(
         file_compression_type: FileCompressionType,
         store: Arc<dyn ObjectStore>,
-    ) {
+    ) -> Result<()> {
         let ctx = SessionContext::new();
         let url = Url::parse("file://").unwrap();
         ctx.runtime_env().register_object_store(&url, store.clone());
         let filename = "1.json";
+        let tmp_dir = TempDir::new()?;
```

Comment:
FWIW it is also fine in test code to use `unwrap()` and avoid having to return `Result`s -- either is acceptable I think.